src/OrchardCore.Modules/OrchardCore.Flows/Controllers/AdminController.cs

```diff
@@ -48,12 +56,41 @@ public async Task<IActionResult> BuildEditor(string id, string prefix, string pr
                 colSize = (int)Math.Round((double)metadata.Size / 100.0 * 12);
 
                 cardCollectionType = nameof(FlowPart);
+
             }
             else
             {
                 cardCollectionType = nameof(BagPart);
             }
 
+            // Get Contained Content Types
+            var contentTypeDefinition = _contentDefinitionManager.GetTypeDefinition(parentContentType);
+            IEnumerable<ContentTypeDefinition> containedContentTypeDefinitions = null;
+
+            foreach (var part in contentTypeDefinition.Parts)
+            {
+                if (string.Equals(part.Name, partName, StringComparison.OrdinalIgnoreCase))
+                {
+                    IEnumerable<string> contentTypes = null;
+
+                    if (part.PartDefinition.Name == nameof(FlowPart))
```

Comment:
This should be via an aspect

Reply:
PartDefinition is already there and I am comparing in same module - the `FlowPart`  class is accessible so i guess `nameof(FlowPart)` is fine here.